packages/components/src/colors.ts

```diff
@@ -9,52 +9,102 @@
  * Copyright (C) The OpenCRVS Authors located at https://github.com/opencrvs/opencrvs-core/blob/master/AUTHORS.
  */
 
-import darken from 'polished/lib/color/darken'
-import lighten from 'polished/lib/color/lighten'
-
-const config = {
-  // Pallete
-  primary: '#4972BB', // indigo
-  secondary: '#4A8AD7', // blue
-  tertiary: '#CCCCCC', // grey
-
-  purple: '#8049B7', // in progress
-  orange: '#F1B162', // ready for review
-  red: '#D53F3F', // requires updates
-  green: '#49b78d', // registered
-  blue: '#4A8AD7', // certified
-  teal: '#4CC1BA', // charts
-  yellow: '#EDC55E', // focus state
-
-  // Status
-  positive: '#49B78D', // green
-  neutral: '#F1B162', // orange
-  negative: '#D53F3F', // red
+export const colors = {
+  // Primary Blue
+  primary: '#0B68F9',
+  primaryDark: '#1447b9',
+  primaryDarker: '#002661',
+  primaryLight: '#C7DDFF',
+  primaryLighter: '#E7F0FC',
+
+  // Secondary Purple
+  purple: '#785AE6',
+  purpleDark: '#45288A',
+  purpleDarker: '#373050',
+  purpleLight: '#D8D3F8',
+  purpleLighter: '#F0ECF9',
+
+  // Secondary Orange
+  orange: '#EA8A25',
+  orangeDark: '#B65618',
+  orangeDarker: '#753919',
+  orangeLight: '#F9DFAF',
+  orangeLighter: '#FEF9EE',
+
+  // Secondary Green
+  green: '#39AB7F',
+  greenDark: '#1D7E5E',
+  greenDarker: '#15503F',
+  greenLight: '#B3E7CE',
+  greenLighter: '#EFFAF5',
+
+  // Secondary Red
+  red: '#D53F3F',
+  redDark: '#B02525',
+  redDarker: '#792323',
+  redLight: '#E79393',
+  redLighter: '#FAE6E6',
+
+  // Secondary Yellow
+  yellow: '#FBD91E',
+  yellowDark: '#E2B605',
+  yellowDarker: '#85530E',
+  yellowLight: '#FEFCC3',
+  yellowLighter: '#FEFDE8',
+
+  // Secondary Teal
+  teal: '#1E7F93',
+  tealDark: '#225968',
+  tealDarker: '#0A3944',
+  tealLight: '#AEDFEA',
+  tealLighter: '#EBF7FA',
+
+  // DEPRECTATED
+  blue: '#4A8AD7',
+  blueDark: '#4079BD',
+  blueDarker: '#3668A3',
+  blueLight: '#72A3E0',
+  blueLighter: '#A0C3EB',
+  secondary: '#4A8AD7',
+  tertiary: '#CCCCCC',
+  subheaderCopyBirth: '#007A5E',
+  subheaderCopyDeath: '#4B5EAA',
+  subheaderCopyMarriage: '#E4BC0C',
+  backgroundPrimary: '#36304E',
```

Comment:
Do we have a plan or a ticket to remove these altogether later, or could they be removed now? We have a tendency to not clean up deprecated items so trying to push it going forwards.

Otherwise the pull request looks good to merge 👍